src/lib/components/button-menu.svelte

```diff
@@ -6,20 +6,23 @@
   export let label = undefined;
   export let disabled = false;
   export let small = false;
+  export let extraClass = "";
+  export let wrapperExtraClass = "";
   let isOpen = false;
 
   function handleClickOutside(_event) {
     isOpen = false;
   }
 </script>
 
-<div class="wrapper">
+<div class={`wrapper ${wrapperExtraClass}`}>
```

Comment:
```suggestion
<div class="wrapper {wrapperExtraClass}">
```
un peu plus digeste déjà!

Reply:
En effet !
Je ne sais pas pourquoi mais 1) je n'ai pas pensé à cette syntaxe - 2) j'ai pensé que ça ne marcherait pas...